lib/arguments/file-url.js

```diff
@@ -4,7 +4,12 @@ import {fileURLToPath} from 'node:url';
 export const safeNormalizeFileUrl = (file, name) => {
 	const fileString = normalizeFileUrl(file);
 
-	if (typeof fileString !== 'string') {
+	const isString = typeof fileString === 'string'
+		// In Deno node:process execPath is a special object, not just a string:
+		// https://github.com/denoland/deno/blob/f460188e583f00144000aa0d8ade08218d47c3c1/ext/node/polyfills/process.ts#L344
+		|| fileString?.__proto__ === String.prototype; // eslint-disable-line no-proto
```

Comment:
```suggestion
		|| Object.getPrototypeOf(fileString) === String.prototype;
```

This avoids the linting warning, and allows running with `node --disable-proto=throw`.

Reply:
Sorry about forgetting to handle `undefined`/`null`, did not think of it. :)